bioimageio/core/_resource_tests.py

```diff
@@ -361,6 +397,50 @@ def _test_expected_resource_type(
     )
 
 
+class PrecisionArgs(TypedDict):
+    """
+    Arguments, both deprecated and current, for setting the precision during validation.
+    """
+
+    absolute_tolerance: float
+    relative_tolerance: float
+    decimal: Optional[int]
+
+
+def _handle_legacy_precision_args(
+    absolute_tolerance: float, relative_tolerance: float, decimal: Optional[int]
+) -> PrecisionArgs:
+    """
+    Transform the precision arguments to conform with the current implementation.
+
+    If the deprecated `decimal` argument is used it overrides the new behaviour with
+    the old behaviour.
+    """
+    # Already conforms with current implementation
+    if decimal is None:
+        return {
+            "absolute_tolerance": absolute_tolerance,
+            "relative_tolerance": relative_tolerance,
+            "decimal": decimal,
+        }
+
+    warnings.warn(
+        "The argument `decimal` has been depricated in favour of "
+        + "`relative_tolerance` and `absolute_tolerance`, with different validation "
+        + "logic, using `numpy.testing.assert_allclose, see "
+        + "'https://numpy.org/doc/stable/reference/generated/"
+        + "numpy.testing.assert_allclose.html'. Passing a value for `decimal` will "
+        + "cause validation to revert to the old behaviour."
+    )
+    # decimal overrides new behaviour,
+    #   have to convert the params to emulate old behaviour
+    return {
+        "absolute_tolerance": 10 ** (-decimal),
```

Comment:
I suppose to resemble previous behavior we need an additional factor of 1.5 (see https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_almost_equal.html)

```suggestion
        "absolute_tolerance": 1.5 * 10 ** (-decimal),
```